src/js/common/schemaform/FormPage.jsx

```diff
@@ -158,7 +172,9 @@ class FormPage extends React.Component {
                 afterText="»"/>
             </div>
           </div>
-          {!form.disableSave && <div className="row">
+          {!form.disableSave && <SaveStatus
```

Comment:
It feels a little weird to me to pass SaveFormLink as a child to SaveStatus. I think I would just pull the row div in here and put them next to each other.

Reply:
Good point, changes made.